### PR TITLE
rome: add 12.1.2

### LINKED ISSUE #5061: [Request]: Biome
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Biome

### Description

Toolchain of the web

### Homepage

https://github.com/biomejs/biome

### Download Link(s)

https://github.com/biomejs/biome/releases/download/cli%2Fv1.0.0/biome-win32-arm64.exe
https://github.com/biomejs/biome/releases/download/cli%2Fv1.0.0/biome-win32-x64.exe

### Some Indication of Popularity/Repute

GitHub nearly 500 stats; Fork of Rome (23k Stars).

### PR DESCRIPTION
**Pull request by CryoRenegade**
rebased to new branch to not get addon commits, still need some help with the checkver and autoupdate if any1 wants.

Closes #5061

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/rome.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "12.1.2",
+    "description": "Formatter, linter, bundler, and more for JavaScript, TypeScript, JSON, HTML, Markdown, and CSS.",
+    "homepage": "https://rome.tools/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/rome/tools/releases/download/cli%2Fv12.1.2/rome-win32-x64.exe",
```
Comment (aliesbelik) [inline bucket/rome.json:8]:
```suggestion
            "url": "https://github.com/rome/tools/releases/download/cli%2Fv12.1.2/rome-win32-x64.exe#/rome.exe",
```

File: bucket/rome.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "508e826da8036eb6eeb25d9377997559b7978fe9a2d676adf3f2918a0a786be1",
+            "bin": [ [ "rome-win32-x64.exe", "rome" ] ]
```
Comment (aliesbelik) [inline bucket/rome.json:10]:
```suggestion

```

File: bucket/rome.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        },
+        "arm64": {
+            "url": "https://github.com/rome/tools/releases/download/cli%2Fv12.1.2/rome-win32-arm64.exe",
```
Comment (aliesbelik) [inline bucket/rome.json:13]:
```suggestion
            "url": "https://github.com/rome/tools/releases/download/cli%2Fv12.1.2/rome-win32-arm64.exe#/rome.exe",
```

File: bucket/rome.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "732bd9c8c13ba758927108a80180b12be9a0f7f7fcc77a2160377e001aa83e46",
+            "bin": [ [ "rome-win32-arm64.exe", "rome"] ]
```
Comment (aliesbelik) [inline bucket/rome.json:15]:
```suggestion

```

File: bucket/rome.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        }
+    },
```
Comment (aliesbelik) [inline bucket/rome.json:17]:
```suggestion
    },
    "bin": "rome.exe",
```

File: bucket/rome.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    "checkver": {
+        "github": "https://github.com/rome/tools"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/rome/tools/releases/download/cli%2Fv$version/rome-win32-x64.exe"
```
Comment (aliesbelik) [inline bucket/rome.json:24]:
```suggestion
                "url": "https://github.com/rome/tools/releases/download/cli%2Fv$version/rome-win32-x64.exe#/rome.exe"
```

File: bucket/rome.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+            },
+            "arm64": {
+                "url": "https://github.com/rome/tools/releases/download/cli%2Fv$version/rome-win32-arm64.exe"
```
Comment (aliesbelik) [inline bucket/rome.json:27]:
```suggestion
                "url": "https://github.com/rome/tools/releases/download/cli%2Fv$version/rome-win32-arm64.exe#/rome.exe"
```

File: bucket/rome.json [hunk 8]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "12.1.3",
+    "description": "Formatter, linter, bundler, and more for JavaScript, TypeScript, JSON, HTML, Markdown, and CSS.",
+    "homepage": "https://rome.tools/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/rome/tools/releases/tag/cli/v12.1.3/rome-win32-x64.exe#/rome.exe",
```
Comment (aliesbelik) [inline bucket/rome.json:8]:
```suggestion
            "url": "https://github.com/rome/tools/releases/download/cli%2Fv12.1.3/rome-win32-x64.exe#/rome.exe",
```

File: bucket/rome.json [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "508e826da8036eb6eeb25d9377997559b7978fe9a2d676adf3f2918a0a786be1",
```
Comment (aliesbelik) [inline bucket/rome.json:9]:
```suggestion
            "hash": "d715c5ea1b7dadfbff37e62969371c495c8072be9165197f69a9ef8965fbb1a7"
```

File: bucket/rome.json [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+        },
+        "arm64": {
+            "url": "https://github.com/rome/tools/releases/tag/cli/v12.1.3/rome-win32-arm64.exe#/rome.exe",
```
Comment (aliesbelik) [inline bucket/rome.json:12]:
```suggestion
            "url": "https://github.com/rome/tools/releases/download/cli%2Fv12.1.3/rome-win32-arm64.exe#/rome.exe",
```

File: bucket/rome.json [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "732bd9c8c13ba758927108a80180b12be9a0f7f7fcc77a2160377e001aa83e46",
```
Comment (aliesbelik) [inline bucket/rome.json:13]:
```suggestion
            "hash": "6378b84f8c432dd56a590085b7bd795e70e2e6b3c5394986fa9b7c5504ee817b"
```

File: bucket/rome.json [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+        }
+    },
+    "bin": "rome.exe",
+    "checkver": {
+        "github": "https://github.com/rome/tools",
+        "regex": "\/releases\/tag\/cli\/(?:v|V)?([\d.]+)"
```
Comment (aliesbelik) [inline bucket/rome.json:19]:
```suggestion
        "regex": "/releases/tag/cli/(?:v|V)?([\\d.]+)"
```

File: bucket/rome.json [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/rome/tools/releases/tag/cli/v$version/rome-win32-x64.exe#/rome.exe"
```
Comment (aliesbelik) [inline bucket/rome.json:24]:
```suggestion
                "url": "https://github.com/rome/tools/releases/download/cli%2Fv$version/rome-win32-x64.exe#/rome.exe"
```

File: bucket/rome.json [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+            },
+            "arm64": {
+                "url": "https://github.com/rome/tools/releases/tag/cli/v$version/rome-win32-arm64.exe#/rome.exe"
```
Comment (aliesbelik) [inline bucket/rome.json:27]:
```suggestion
                "url": "https://github.com/rome/tools/releases/download/cli%2Fv$version/rome-win32-arm64.exe#/rome.exe"
```

**Comment by CryoRenegade** [comment]
Thank you @aliesbelik ... regex always kicks my butt.

**Pull request merged**